app/models/asset_field_type.rb

```diff
@@ -52,6 +52,15 @@ def mapping
 
   private
 
+  def is_image?(mime_type)
+    ['image/jpeg', 'image/pjpeg', 'image/png','application/pdf' ,'image/x-png', 'image/gif'].include?(mime_type)
```

Comment:
I actually made this Method to check in
```ruby
 def metadata=(metadata_hash)
    @metadata = format_data(metadata_hash).deep_symbolize_keys
    @existing_data = metadata_hash[:existing_data]
    Paperclip::HasAttachedFile.define_on(self.class, :asset, existing_metadata)
end
```
To change `image_styles` to `styles` so that if the asset is an image or pdf that it can run post processing. Originally I tried `image?` but that relies on `asset_content_type` being undefined at that moment.